tools/sdk-cli/src/commands.rs

```diff
@@ -46,6 +46,9 @@ pub(crate) enum Commands {
     /// Sync local data with remote node
     Sync {},
 
+    /// Triggers a backup of the local data
+    Backup {},
```

Comment:
Nit: maybe this should be called `BackupRestore`? Because it doesn't just backup local data to the GL node, but also restores new data from GL to the local node's DB.